doc/user/content/administration/billing.md

```diff
@@ -0,0 +1,113 @@
+---
+title: "Usage & billing"
+description: "Understand the billing model of Materialize, and learn best practices for cost control."
+menu:
+  main:
+    parent: "manage"
+    weight: 50
+---
+
+{{< note >}}
+Access to usage and billing requires **administrator** privileges.
+{{</ note >}}
+
+Materialize determines billing based on your storage and compute usage.
```

Comment:
I think we can pull some info from the longer Note later on to these first few sentences. Maybe something like:

> Materialize bills per second based on the cluster, or clusters, you provision to power your workloads. Each cluster is a pool of resources (CPU, memory, and scratch disk space) that must stay up and running to continually provide you with always-fresh results. The cluster size needed for any given workload will depend on that specific workload's Compute and Storage requirements.